main.yml

```diff
@@ -331,6 +332,17 @@ subscriptions:
       - /staging/subscriptions
   git_repo: https://github.com/RedHatInsights/curiosity-frontend
 
+topological-inventory:
+  title: Topological inventory
```

Comment:
Inventory should probably be capitalized

Reply:
Shouldn't all things in PF4 sentence case?

Reply:
I don't know what you mean - what does PF4 have to do with this? All other apps have their titles in title case. If they're specifically telling you they want it to be sentence case, I can approve it, but it's going to look extremely weird in the sidenav when everything else is capitalized.

Reply:
No. Titles are capitalized since they're proper. If that has changed, then all of them should be updated. My new mockups with landing page stuff had every words caps.

Reply:
Well PF4 defined that everything should be sentence case, but yeah other apps has all upper case, I'll update it.

Reply:
Cool. If we decide to start following those guidelines, we can make a change for all of them at once.

Reply:
@karelhala Not "everything". As per their guidelines:
```
Sentence style

Use sentence-style capitalization for all text elements, except:

    Product names and proper nouns
```